### PR TITLE
[SMALLFIX] default acl should inherit from access acl entries when first created

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

**Comment by yuzhu** [comment]
@gpang PTAL

**Comment by gpang** [comment]
@yuzhu can you add a unittest for this?

**Comment by yuzhu** [comment]
@gpang added an integration test, PTAL.

**Comment by yuzhu** [comment]
jenkins, test this please

**Comment by gpang** [comment]
Jenkins, test this please

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java [hunk 1]
```diff
@@ -3256,7 +3256,7 @@ private void setAclSingleInode(RpcContext rpcContext, SetAclAction action,
         targetInode.removeExtendedAcl();
         break;
       case REMOVE_DEFAULT:
-        targetInode.setDefaultACL(new DefaultAccessControlList());
```
Comment (gpang) [inline core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java]: When is this empty constructor ever useful?

Comment (yuzhu) [reply core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java]: it is useful when the client tries to deserialize the `DefaultAccessControlList` from the network

**Comment by yuzhu** [comment]
@gpang comment addressed, PTAL

**Pull request merged**